crates/holochain_types/src/dna/dna_file.rs

```diff
@@ -87,11 +87,23 @@ impl DnaFile {
     }
 
     /// Hot swap coordinator zomes for this dna.
```

Comment:
We still have code comments referencing Hot swap. Might want to change these to Update Coordinators as well. I saw some tests as well that have the term Hot swap in there.